agent/core/src/main/java/org/jolokia/restrictor/RestrictorFactory.java

```diff
@@ -45,18 +71,19 @@ public static Restrictor createRestrictor(Configuration pConfig, LogHandler logH
             return customRestrictor;
         }
 
-        String location = NetworkUtil.replaceExpression(pConfig.get(ConfigKey.POLICY_LOCATION));
+        String rawLocation = pConfig.get(ConfigKey.POLICY_LOCATION);
+        String location = NetworkUtil.replaceExpression(rawLocation);
         try {
             Restrictor ret = RestrictorFactory.lookupPolicyRestrictor(location);
             if (ret != null) {
-                logHandler.info("Using policy access restrictor " + location);
+                logHandler.info("Using policy access restrictor " + rawLocation);
```

Comment:
Because it's printed into the logs and would contain - at least in the Basic-Auth case - sensitive information which you don't wan't spread out into your central logging system for public consumption.

But yap, I didn't think about the non-sensitive case. Namely status-quo before.  No quick solution in my head for the moment.